Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -238,6 +260,24 @@ end CircleIntegrable
 theorem circleIntegrable_zero_radius {f : ℂ → E} {c : ℂ} : CircleIntegrable f c 0 := by
   simp [CircleIntegrable]
 
+/-- Circle integrability is invariant when functions change along discrete sets. -/
+theorem CircleIntegrable.congr_of_codiscreteWithin {c : ℂ} {R : ℝ} {f₁ f₂ : ℂ → ℂ}
```

Comment:
```suggestion
theorem CircleIntegrable.congr_codiscreteWithin {c : ℂ} {R : ℝ} {f₁ f₂ : ℂ → ℂ}
```
Given that all the other theorems don't have an `of` there, let's remove this one (and the next one). Sorry I've changed my mind!